<?xml version="0.0"?>
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="ackermann_orthexp">
                <param name="k" value="2.0" />

                <param name="look_ahead_dist" value="1.0" />
                <param name="k_o" value="0.0" />
                <param name="k_g" value="0.5" />
                <param name="k_w" value="0.0" />
                <param name="k_curv" value="0.0" />
                <param name="obst_threshold" value="3.0" />
            </group>
        </group>
    </group>  
</launch>
